ibis/backends/tests/test_aggregation.py

```diff
@@ -1524,8 +1520,9 @@ def test_grouped_case(backend, con):
 
 
 @pytest.mark.notimpl(
-    ["datafusion", "mssql", "polars", "exasol"], raises=com.OperationNotDefinedError
+    ["datafusion", "mssql", "polars"], raises=com.OperationNotDefinedError
 )
+@pytest.mark.notimpl(["exasol"], raises=ExaQueryError)
```

Comment:
Just a nit and unrelated to this pr is that the exception name `ExaQueryError` is not consistent with the rest of the error types having full backend name as the prefix, so I think it should be `ExasolQueryError`